packages/babel-traverse/src/index.js

```diff
@@ -33,6 +33,7 @@ traverse.explode = visitors.explode;
 traverse.NodePath = require("./path");
 traverse.Scope    = require("./scope");
 traverse.Hub      = require("./hub");
+traverse.cache    = require("./cache");
```

Comment:
Do we need to expose it at all, or could we call it automatically in `addAst` when a transform pass starts?